src/test/java/com/itschool/eventmanagment/integration/tests/EventControllerTest.java

```diff
@@ -0,0 +1,53 @@
+package com.itschool.eventmanagment.integration.tests;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.itschool.eventmanagment.models.dtos.EventDTO;
+import com.itschool.eventmanagment.models.entities.User;
+import com.itschool.eventmanagment.repositories.UserRepository;
+import jakarta.transaction.Transactional;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.time.LocalDateTime;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+
+@Slf4j
+@ExtendWith(SpringExtension.class)
+@SpringBootTest
+@AutoConfigureMockMvc
+@Transactional
+@AutoConfigureTestDatabase
+public class EventControllerTest {
+    @Autowired
+    private MockMvc mockMvc;
+    @Autowired
+    private ObjectMapper objectMapper;
+    @Autowired
+    private UserRepository userRepository;
+
+    @Test
+    void testCreateEventShouldPass() throws Exception{
+        LocalDateTime eventDate = LocalDateTime.now();
+        User user = new User();
+        user.setEmail("alina@");
+        user.setFirstName("alina");
+        user.setLastName("miha");
+        userRepository.save(user);
+        EventDTO eventDTO = new EventDTO( "music", "Botosani", eventDate, 1L, 10);
+        mockMvc.perform(post("/api/events").contentType(MediaType.APPLICATION_JSON)
+                .content(objectMapper.writeValueAsString(eventDTO))
+        ).andExpect(status().isOk());
+
```

Comment:
Please consider removing this empty row and maybe adding one after line 47 in order to have events inside the test a bit more decoupled.
Also, for visual improvement, chained methods from mockMvc should be one under the other.